frontend/libs/ui-modules/src/modules/import-export/hooks/export/useExportFieldSelection.ts

```
@@ -5,15 +5,17 @@ import { TExportHeader } from '../../types/export/exportTypes';
   onConfirm,
   onOpenChange,
 }: {
   entityType: string;
+  filters?: Record<string, any>;
   onConfirm: (selectedFields: string[]) => void;
   onOpenChange: (open: boolean) => void;
 }) => {
   const { data, loading } = useQuery(GET_EXPORT_HEADERS, {
-    variables: { entityType },
+    variables: { entityType, ...(filters ? { filters } : {}) },
     skip: !open,
```

ESLint

[error] Unexpected use of 'open'.

(no-restricted-globals)